src/renderer/component/errors/errors.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Errors = props =>
+  <div className="errors">
+    <ul className="errors-list">
+      <li className="errors-list__item">
+        {props.errors.map(error => <p>{error}</p>)}
+      </li>
```

Comment:
Could you map the errors to `li` here instead of `p`? This doesn't really make sense semantically otherwise.

Reply:
Definitely, and good flag.